docs/About/release_policy.md

```
@@ -0,0 +1,226 @@
+---
+
+# Godot release policy
+
+Godot's release policy is in constant evolution. The description below
+provides a general idea of what to expect, but what will actually
+happen depends on the choices of core contributors and the needs of the
+community at a given time.
+
+## Godot versioning
+
+Godot loosely follows [Semantic Versioning ](https://semver.org/)_ with a
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)